test_scripts/RC/OnRemoteControlSettings/023_Release_resource_on_PTU_with_app_revoked.lua

```diff
@@ -0,0 +1,45 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/10
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/resource_allocation.md
+-- Item: Use Case 3: Excpetion 2.1
+--
+-- Requirement summary:
+-- [SDL_RC] Resource allocation based on access mode
+--
+-- Description:
+-- In case:
+-- Any trigger of Policy Table Update happened and in received PTU RC_app_1 is revoked
+--
+-- SDL must:
+-- 1) SDL releases module_1 from RC_app_1 control
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonRC = require('test_scripts/RC/commonRC')
+local json = require("modules/json")
+
+
+--[[ Local Functions ]]
+local function ptu_update_func(tbl)
+  tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID] = json.null
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonRC.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", commonRC.start)
+runner.Step("RAI1, PTU with RADIO for App1", commonRC.rai_ptu)
+
+runner.Title("Test")
+runner.Step("Enable RC from HMI with AUTO_DENY access mode", commonRC.defineRAMode, { true, "AUTO_DENY"})
+runner.Step("Activate App1", commonRC.activate_app)
+-- App1: FULL
+runner.Step("Module RADIO App1 ButtonPress allowed", commonRC.rpcAllowed, { "RADIO", 1, "ButtonPress" })
+runner.Step("RAI2, PTU without RADIO for App1", commonRC.rai_ptu_n, { ptu_update_func, 2 })
+runner.Step("Module RADIO App1 SetInteriorVehicleData disallowed", commonRC.rpcDenied, { "RADIO", 2, "SetInteriorVehicleData", "DISALLOWED"})
+runner.Step("Activate App2", commonRC.activate_app, { 2 })
+-- App1: BACKGROUND, App2: FULL
+runner.Step("Module RADIO App2 SetInteriorVehicleData allowed", commonRC.rpcAllowed, { "RADIO", 2, "SetInteriorVehicleData"})
+
+runner.Title("Postconditions")
+runner.Step("Stop SDL", commonRC.postconditions)
```

Comment:
@aderiabin It would be better if we have an additional verification that both modules are not available. Meaning not just a single one.

Reply:
@dboltovskyi Additional checks were added (see 66f15f8)